app/bundles/NotificationBundle/Controller/JsController.php

```diff
@@ -24,9 +24,10 @@ class JsController extends CommonController
      */
     public function manifestAction()
     {
+        $gcm_sender_id = $this->factory->getParameter('gcm_sender_id', '446150739532');
```

Comment:
Per Symfony coding standards, variables should be camel case (`$gcmSenderId`). Also, `$this->factory` is deprecated and thus we're not using it in new code. Please change that to `$this->get('mautic.helper.core_parameters')->getParameter('gcm_sender_id', '446150739532');`